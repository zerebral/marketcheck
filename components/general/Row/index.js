import styled from 'styled-components'
import { rem, clearFix } from 'polished'
import { grid } from '%/styles'

const gutter = rem(grid.gutter / 2)

const Row = styled.div`
  margin-left: -${gutter};
  margin-right: -${gutter};

  ${clearFix()}
`

export default Row
