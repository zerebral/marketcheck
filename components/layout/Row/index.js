import styled from 'styled-components'
import { rem, clearFix } from 'polished'
import { grid } from '%/styles'

const gutter = rem(grid.gutter / 2)

const Row = styled.div`
  ${clearFix()}
  margin: 0 -${gutter};
`

export default Row
