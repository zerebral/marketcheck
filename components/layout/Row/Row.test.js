/* eslint-env jest */
import React from 'react'
import { shallow } from 'enzyme'
import 'jest-styled-components'

import Row from './index'

describe('Row Component', () => {
  test('Row Passes shallow snapshot', () => {
    const wrapper = shallow(<Row />)
    expect(wrapper).toMatchSnapshot()
  })
})
