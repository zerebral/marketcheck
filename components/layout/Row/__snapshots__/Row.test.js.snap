// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Row Component Row Passes shallow snapshot 1`] = `
.c0 {
  margin: 0 -0.625rem;
}

.c0::after {
  clear: both;
  content: "";
  display: table;
}

<div
  className="c0"
/>
`;
